specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/file.json

```diff
@@ -798,6 +842,55 @@
           "readOnly": true,
           "description": "The approximate size of the data stored on the share. Note that this value may not include all recently created or recently resized files."
         },
+        "leaseStatus": {
```

Comment:
After Acquire lease on a share, then get the share, the LeaseDuration, LeaseState, LeaseStatus seems won't return.

Reply:
Fixed.